docs/trace/exception-handling/README.md

```diff
@@ -1,8 +1,8 @@
-# Exception Handling
+# Exception Reporting
```

Comment:
we could rename the whole folder/project to Exception Reporting, instead of Exception handling.

Reply:
Either way sounds okay to me - just make sure the folder name is consistent with the title.